src/systems/systems/src/landing_gear/mod.rs

```diff
@@ -744,6 +756,131 @@ impl LandingGearControlInterfaceUnit {
     fn status(&self) -> LgciuStatus {
         self.status
     }
+
+    fn discrete_word_1(&self) -> Arinc429Word<f32> {
+        if !self.is_powered {
+            Arinc429Word::new(0., SignStatus::FailureWarning)
+        } else {
+            let mut word = Arinc429Word::new(0., SignStatus::NormalOperation);
+            set_arinc429_bit(
+                &mut word,
+                11,
+                !self.sensor_inputs.left_gear_up_and_locked && !self.gear_handle_is_down(),
```

Comment:
I'm wondering if we should use sensor_inputs.left_gear_up_and_locked directly or use access methods that are also handling the value if failed/unpowered. Right now probably won't change anything as you just send invalid frame when unpowered, but something to remember to check when adding more things I believe.